arecord: add example list audio devices and intractive interface (#4530)
tldr-pages/tldr: pages/linux/arecord.md

- Record a snippet and save it as mp3 (finish with Ctrl-C when done):

`arecord -vv --format=cd --file-type raw | lame -r - {{path/to/file.mp3}}`

> - List all sound cards and digital audio devices

Reviewer: ```suggestion
- List all sound cards and digital audio devices:
```

Apparently this is needed for formatting

> `arecord -l`

Reviewer 2: Please use long argument format:
```suggestion
`arecord --list-devices`
```

> - Allow intractive interface like play/pause via spacebar/enter

Reviewer: ```suggestion
- Allow intractive interface like play/pause via spacebar/enter:
```
This also

> - Allow intractive interface like play/pause via spacebar/enter:

Reviewer 2: ```suggestion
- Allow interactive interface (e.g. use space-bar or enter to play or pause):
```

> `arecord -i`

Reviewer 2: ```suggestion
`arecord --interactive`
```
